lte/gateway/c/core/oai/common/common_types.h

```diff
@@ -391,6 +402,9 @@ typedef struct {
   apn_config_profile_t apn_config_profile;
   rau_tau_timer_t rau_tau_timer;
   charging_characteristics_t default_charging_characteristics;
+#define MAX_REGIONAL_SUB 10
```

Comment:
looks like you are using `MAX_REGIONAL_SUB` in multiple places, then move it to the top portion of the header file.

Reply:
1. MAX_REGIONAL_SUB is the maximum zonecodes that can be sent from HSS. 
2. MAX_TACS_PER_SAC is the maximum number of TACs in the TAC list per SAC.  
3. MAX_ZONE_CODE_LEN is the size of zone_code string. I will rename this as MAX_ZONE_CODE_STR_LEN for better readability